rer/subsites/upgrades/upgrades.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+default_profile = 'profile-rer.erpolicy:default'
+
+
+def to_2100(context):
+    """
+    """
+    logger.info('Upgrading rer.erpolicy to version 2100')
```

Comment:
Minor confusion here (copy-paste? 😁 ) Should probably be `rer.subsites` instead of `rer.erpolicy`.